packages/search-adapter/src/empathy/mappers/request/params/empathy-request-related-tags-query.mapper.ts

```diff
@@ -8,9 +9,24 @@ import { RequestMapper } from '../../../empathy-adapter.types';
  * @public
  */
 @injectable()
-export class EmpathyRequestRelatedTagsQueryMapper implements RequestMapper<QueryableRequest, string> {
-
+export class EmpathyRequestRelatedTagsQueryMapper
+  implements RequestMapper<QueryableRequest, string>
+{
   map({ relatedTags = [] }: QueryableRequest, query: string): string {
-    return relatedTags.reduce((chain, rt) => `${chain} ${rt.tag}`, query).trim();
+    const [leftRts, rightRts] = this.splitRelatedTagsByQueryPosition(relatedTags, query);
```

Comment:
There is a warning here because the IDE is complaining about the possible undefined of the `relatedTags`.

It's not a big deal but you can remove if adding

```javascript
  if (!relatedTags ){
      return query;
    }
```